docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  TODO
+  (_operatingSystemName_) is not a supported platform for _componentName_. Skipping _pluginid_... Supported: _requiredOperatingSystem_
 
   **Action:**
 
-  TODO
+  Contact the author of the component, or a system administrator.
 
 
 
 ### ZWED0154E":"(%s) is not a supported architecture for %s. Skipping (%s)... Supported: %s
 
   **Reason:**
 
-  TODO
+  (_operatingSystemName_) is not a supported platform for _componentName_. Skipping _pluginid_... Supported: _requiredOperatingSystem_
 
   **Action:**
 
-  TODO
+  Contact the author of the component, or a system administrator.
 
 
 
 ### ZWED0155E":"(%s) is not a supported endpoint for %s. Skipping (%s)... Supported: %s
 
   **Reason:**
 
-  TODO
+  (_url_) is not a supported endpoint for _componentName_. Skipping _pluginid_... Supported: _urls_
 
   **Action:**
 
-  TODO
+  Refer to component API doc.
 
 
 
 ### ZWED0156E":"Could not register default plugins into app-server
 
   **Reason:**
 
-  TODO
+  Could not register default plugins into app-server
 
   **Action:**
 
-  TODO
+  Verify integrity of required plugin files, or contact publisher.
 
 
 
 ### ZWED0157E":"Could not register default plugin %s into app-server
 
   **Reason:**
 
-  TODO
+  Could not register default plugin _pluginid_ into app-server
 
   **Action:**
 
-  TODO
+  Verify integrity of required plugin files, or contact publisher.
 
 
 
 ### ZWED0158E":"Could not listen on address %s:%s. Insufficient permissions to perform port bind."
 
   **Reason:**
 
-  TODO
+  Could not listen on address (_ipAddress_):(_port_). Insufficient permissions to perform port bind.
 
   **Action:**
 
-  TODO
+  Contact system administrator.
```

Comment:
would we see an exception above this message? perhaps they need to give the admin that message.
perhaps the port isnt valid, the ip isnt valid, or both of them are correct but the zowe runtime user just lacks permission.